bin/setup.sh

```diff
@@ -1,90 +1,94 @@
 #!/bin/bash
 ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")" && cd .. && pwd)
-
-TITLE="WP-Docker-Construct"
-ADMIN_EMAIL="engenharia@log.pt"
-REPOSITORY="git@github.com:log-oscon/WP-Construct.git"
+cd ROOT
 
 echo "-----------------------------"
-echo "${TITLE}"
+echo "${PWD##*/}"
 echo "-----------------------------"
 
-read -p "Write your domain: " site_url
-URL="${site_url// /-}.local";
+site_url='localhost'
+read -p "Write your domain | default: ${site_url} |: " site_url
+URL=${site_url// /-};
 
-read -p "is Multisite (y/n)? default n: " is_multisite
-case "$is_multisite" in 
-  y|Y ) MULTISITE=1;;
-  n|N ) MULTISITE=0;;
-  * ) MULTISITE=0;;
-esac
-
-## CHECKOUT PROJECT ##
-if [ ! -z "$REPOSITORY" ] && [ ! -d "./wordpress/.git" ]; then
-	echo "No local repository found. Downloading boilerplate / project from GIT..."
-  cd "$ROOT/wordpress"
-	git clone $REPOSITORY .
-  cd "$ROOT"
+if ! echo "${site_url}" | grep '^[a-zA-Z0-9]*\.[a-zA-Z]*$' >/dev/null; then
+	URL="${URL}.local";
 fi
 
-## PROJECT DEPENDENCIES ##
-echo "Searching for project build script..."
-if [ -f "./wordpress/.scripts/build.sh" ]; then
-	cd "$ROOT/wordpress"
-	echo " * Starting project build"
-	sh .scripts/build.sh
-else
-	echo " * No build (.scripts/build.sh) script found."
-fi
+while true; do
+
+	read -p "Write your admin email: " ADMIN_EMAIL
+
+	if echo "${ADMIN_EMAIL}" | grep '^[a-zA-Z0-9]*@[a-zA-Z0-9]*\.[a-zA-Z]*$' >/dev/null; then
+		break
+	else
+		echo "Please write a valid email."
+	fi
+
+done
+
+new_database() {
+
+	if [ -f "${ROOT}/wordpress/wp-config.php" ]; then
+		docker-compose exec --user www-data phpfpm wp db create --url="${URL}" --path="${ROOT}/wordpress"
+	else
+		echo 'No config file found.'
+	fi
+}
+
+read -p "Wordpress repository, empty to clean wordpress: " REPOSITORY
 
 ## WORDPRESS SETUP ##
-if [ -f "./wordpress/wp-config.php" ]; then
-	echo "WordPress config file found."
+if [ ! -z "$REPOSITORY" ]; then
```

Comment:
You need to check if a repo already exists, with ` && [ ! -d "./wordpress/.git" ]`